src/editor.html

```diff
@@ -79,6 +79,21 @@ <h5 class="section-heading">Display Settings</h5>
 				</div>
 			</div>
 
+			<!--Metric Display Regex-->
+			<div class="gf-form">
+				<span class="gf-form-label width-12">
+				  Value Regex
+				  <info-popover mode="right-normal">
+					  A regex which will decide the part of the value to be displayed. <br/>
+					  In case the regex is empty or it doesn't match any part of the metrics value, all the metric value will be displayed.
+				  </info-popover>
+				</span>
+				<input type="text" name="valueDisplayRegex" ng-model="measurement.valueDisplayRegex" class="gf-form-input max-width-12" ng-change="ctrl.onRender()" />
+				<div ng-if="measurement.valueDisplayRegexValidated == false" style="color: red; padding-left: 5px">
```

Comment:
You could just do the validation here without using the `valueDisplayRegexValidated` variable